independent-publisher-connectors/Mensagia/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@AfriSR The `settings.json` is not a required file to open source, please remove it from your PR.